js/kucoin.js

```diff
@@ -422,7 +422,11 @@ module.exports = class kucoin extends Exchange {
     }
 
     async loadTimeDifference (params = {}) {
-        const response = await this.publicGetTimestamp (params);
+        const method = this.getSupportedMapping (this.id, {
+            'kucoin': 'publicGetTimestamp',
+            'kucoinfutures': 'futuresPublicGetTimestamp',
```

Comment:
This should also be renamed to `'futures'` by market type or by defaultType if we inherit one exchange from another.